langext/include/macros.h

```diff
@@ -89,3 +89,9 @@ typedef enum {
         } \
         return new_array; \
     }
+
+#ifdef RAGE_DISABLE_ANON_UNIONS
+#define RAGE_ANON_UNION(t, v) t v
```

Comment:
`ANON_UNION` is a bit broad a name, given that we're using this anonymous union for the specific purpose of conveniently embedding one struct within another. How about calling it `RAGE_EMBED_STRUCT`?